domovoi.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python3
+
+import argparse
+import logging
+import os
+import shlex
+import sys
+import time
+
+import settings
+
+from solar_car_process import SolarCarProcess
+
+
+class Domovoi:
+    def parse_file(self, processes_file):
+        with open(processes_file) as file:
+            return([SolarCarProcess(shlex.split(path)) for path in file.read().splitlines()])
+
+    def run(self, processes_file):
+        solar_car_processes = self.parse_file(processes_file)
+        # Check processes and respond accordingly
+        while True:
+            for solar_car_process in solar_car_processes:
+                if not solar_car_process.process:
+                    solar_car_processes.remove(solar_car_process)
```

Comment:
Instead of not running that specific process, we probably want to crash the whole program.